samples/bundle-war-deploy/README.md

```diff
@@ -0,0 +1,45 @@
+# Bundle war deploy sample
+This sample demonstrates how you can add to an existing pom.xml of a war project that already exists.
+Everything within the demo-war directory would be an existing Maven project. The additional step to publish this to CICS is the additional plugin in the pom.xml: 
+
+## Set Up
+### Have your system programmer create your bundle definition in CSD
+Your system programmer should create a bundle definition in CSD and tell you the CSD group and BUNDDEF name they have used.
```

Comment:
I think it should be Bundle definition again, not BUNDDEF because BUNDDEF is a CPSM thing.